edx_django_utils/monitoring/internal/code_owner/middleware.py

```diff
@@ -21,10 +21,8 @@ class CodeOwnerMonitoringMiddleware:
 
     Custom attributes set:
     - code_owner: The owning team mapped to the current view.
-    - code_owner_mapping_error: If there are any errors when trying to perform the mapping.
```

Comment:
**Reviewer Note**: This simply didn't exist any more, so this was just clean up.